### PR TITLE
feat(origin-backend): added self ownership flag for organization

### PR DESCRIPTION
**Pull request by ioncreature**

### CODE REVIEW

File: packages/origin-backend/src/pods/organization/dto/full-organization-info.dto.ts [hunk 1]
```diff
@@ -60,6 +60,11 @@ export class FullOrganizationInfoDTO extends PublicOrganizationInfoDTO {
     @IsOptional()
     blockchainAccountSignedMessage?: string;
 
+    @ApiPropertyOptional({ type: Boolean })
+    @IsString()
```
Comment (alexworker23) [inline packages/origin-backend/src/pods/organization/dto/full-organization-info.dto.ts]: `IsBoolean()`

**Comment by alexworker23** [comment]
I receive 403 status when trying to update `selfOwnership` status: 
Screenshot 2021-12-07 at 16 18 54

**Comment by ioncreature** [comment]
> I receive 403 status when trying to update `selfOwnership` status: Screenshot 2021-12-07 at 16 18 54
> 

It allows organization admins only to change self ownership flag only

File: packages/origin-backend/src/pods/organization/dto/full-organization-info.dto.ts (hunk 1 above)
Comment (ioncreature) [reply packages/origin-backend/src/pods/organization/dto/full-organization-info.dto.ts]: thanks, done

**Review by alexworker23 — APPROVED** [review]

**Pull request merged**